core/src/test/java/fr/sncf/osrd/new_infra_state/StandaloneSignalingSimulationTests.java

```diff
@@ -108,7 +110,7 @@ public void lineInfraEnvelopeTests() throws Exception {
         var signalizationEngine = SignalizationEngine.from(infra, infraState);
         var events = StandaloneSignalingSimulation.run(path, infraState, signalizationEngine, envelope);
         for (var e : events)
-            assertEquals(e.position() / speed, e.time());
+            assert e.position() / speed >= e.time();
```

Comment:
you cannot safely use assert in tests. asserts are ignored unless in explicitly enabled, and this includes test asserts too.
this is very important, as we want to be able to run and test the application with asserts disabled